# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome14/gconf110/fix-bdb-backend.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./backends/bdb.c.vanilla	2005-12-12 21:48:59.000000000 +0100
+++ ./backends/bdb.c	2005-12-12 21:52:20.000000000 +0100
@@ -331,7 +331,7 @@
 
 /* } */
 
-static guint32
+guint32
 get_dir_id (BDB_Store * bdb, const char *dir)
 {
   int ret;
@@ -405,7 +405,7 @@
 
 /* { Functions to create directories and maintain the dir hierarchy */
 
-static guint32
+guint32
 get_or_create_dir (BDB_Store * bdb, const char *dir)
 {
   guint32 parent_id;
